queries/selectors.yml

```
@@ -28,6 +28,9 @@ selectors:
           value: tecnologia_servico
         - method: path
           value: models/subsidio
+        - exclude:
+          - method: fqn
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)